examples/plot_example_workflow.py

```diff
@@ -0,0 +1,151 @@
+# the number of epochs marked as bad by autoreject substantially.
+#
+# When making this decision to filter the data, we do want to be careful
+# because filtering can spread sharp, high-frequency transients and
+# distort the phase of the signal. Most evoked response potential
+# analyses use filtering since the interest is in the time series, but
+# if you are doing a frequency based analysis, filtering before the
+# Fourier transform could potentially be avoided by detrending instead.
+
+raw.filter(l_freq=1, h_freq=None)
+epochs = mne.Epochs(raw, events, tmin=0, baseline=None, preload=True)
+ar = autoreject.AutoReject(n_interpolate=[1, 2, 3, 4], random_state=11,
+                           n_jobs=1, verbose='tqdm')
+ar.fit(epochs[:20])  # fit on the first 20 epochs to save time
+epochs_ar, reject_log = ar.transform(epochs, return_log=True)
+
+# visualize the dropped epochs
+epochs[reject_log.bad_epochs].plot(scalings=dict(eeg=1e-4))
+
+# %%
+# Finally, we can apply independent components analysis (ICA) to remove
+# eyeblinks from the data. If our analysis were to be very dependent on
+# sensors at the front of the head or frequency components near the
+# frequency of eyeblink potentials or if we wanted to be extra careful
+# or if we had lots of data and didn't want to go through the trouble,
+# we could skip ICA and use the previous result. However, ICA can increase
+# the amount of usable data and is one of the most commonly used methods
+# in EEG analyses.
+#
+# We can see in the plots below that ICA effectively removed eyeblink
+# artifact, and, a bit counterintuitively, in doing so caused a few more
+# epochs to be dropped. This is because the threshold was no longer biased
+# upwards by the eyeblinks and so was more stringent.
+#
+# These are the basic steps for a workflow with decisions that must be
+# made based on what the data is being used for. Following this may help
+# you optimize your use of ``autoreject`` in preprocessing.
+
+ica = mne.preprocessing.ICA(random_state=99)
+reject = autoreject.get_rejection_threshold(epochs)
+ica.fit(epochs, reject=reject)
+# get the rejected components
+
+# plot with and without eyeblink component
+ica.plot_overlay(epochs.average(), exclude=ica.exclude)
+ica.apply(epochs, exclude=ica.exclude)
+
+# run ``autoreject`` one more time
+ar = autoreject.AutoReject(n_interpolate=[1, 2, 3, 4], random_state=11,
+                           n_jobs=1, verbose='tqdm')
+ar.fit(epochs[:20])  # fit on the first 20 epochs to save time
+epochs_ar, reject_log = ar.transform(epochs, return_log=True)
+epochs[reject_log.bad_epochs].plot(scalings=dict(eeg=1e-4))
+
+# visualize the final outcome of the data
+fig, axes = plt.subplots(2, 1, figsize=(6, 6))
+ylim = dict(eeg=(-5, 5))
+epochs.average().plot(exclude=[], axes=axes[0], ylim=ylim, show=False)
+axes[0].set_title('Before autoreject')
+epochs_ar.average().plot(exclude=[], axes=axes[1], ylim=ylim)
+axes[1].set_title('After autoreject')
```

Comment:
When I plot this, I only see the "before autoreject" and "EEG 32 channels" as the title one below the other .....not "after autoreject"

Reply:
See here https://758-59485140-gh.circle-artifacts.com/0/html/auto_examples/plot_example_workflow.html#sphx-glr-auto-examples-plot-example-workflow-py, that's how it will look upon merging